module2/lessons/advanced_routing.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Advanced Routing
+layout: page
+---
+## Learning Goals
+
+- Understand what the `resources` syntax in `routes.rb` generates for us.
+- Understand what nesting `resources` in `routes.rb` generates for us.
+- Understand the 5 pieces of information `rails routes` gives us.
+- Use route helpers
+- Understand the difference between namespacing and nested `resources``
+
+## Vocabulary
+
+- routes
+- route helper
+- namespace
+
+## Set Up
+For part of this lesson we'll use the [`advanced-routing`](https://github.com/turingschool-examples/set-list-api/tree/advanced-routing-start) branch of the Set List Tutorial.
```

Comment:
Just update branch name to match `advanced-routing-start` bc I did a bad job naming things exactly how they were named in our old repo lol